phoenix-postgres-pgadmin/docker-compose.yaml

```diff
@@ -0,0 +1,56 @@
+version: '3.7' 
+ 
+services: 
+ 
+  example_app:
+    build: example_app
+    environment:  
+      - POSTGRES_USER=postgres 
+      - POSTGRES_PASSWORD=postgres 
+      - POSTGRES_DB=ExampleAppDB 
+      - PGHOST=db 
+      - PGPORT=5432 
+      - HOST=localhost
+      - PORT=4000
+    depends_on: 
+      - db 
+    ports:  
+      - "4000:4000" 
+
+  db: 
+    image: postgres 
```

Comment:
Can you avoid using the latest version and fix it, with the `postgres:12.2` for example, to avoid future issues (ie: breaking changes in the next major version ...)

Reply:
Got it! I add the latest version as you suggested!